scripts/publish.sh

```diff
@@ -30,5 +45,12 @@ rm public/documentation/index.xml
 rm public/community/index.xml
 rm public/development/index.xml
 
+DESTINATION_STABLE_VERSION="public/documentation/stable"
+echo "Copying the stable documentation from ${STABLE_VERSION_FOLDER} to ${DESTINATION_STABLE_VERSION}"
+cp -r ${STABLE_VERSION_FOLDER} ${DESTINATION_STABLE_VERSION}
+
+echo "Add noindex meta tag to all HTML files in public/documentation/[0-9]*"
+find public/documentation/ -type f -regex "public/documentation/[0-9].*" -name "*.html" | xargs  sed -i '/<\/head>/i\<meta name="robots" content="noindex" \/>'
```

Comment:
This SO post provides a good answer: http://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux. We'd only have to remove the backup files.